modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -659,6 +667,8 @@ pub struct Wallet {
     cfg: WalletConfig,
     secp: Secp256k1<All>,
     btc_rpc: DynBitcoindRpc,
+    /// The result of last successful get_block_count
+    block_count_local: std::sync::Mutex<Option<u32>>,
```

Comment:
Why not [`tokio::sync::watch::Receiver`](https://docs.rs/tokio/latest/tokio/sync/watch/struct.Receiver.html)?

Reply:
Overkill for updating/reading an u32. I was going to go with `AtomicU32`, but then I need to use a magic number for none, and I'm superstitious.